tests/testcases/test_presets.py

```diff
@@ -172,7 +172,7 @@ def test_find_newest_preset_1(self):
         path = os.path.join(PRESETS, "Bar Device", "picture.png")
         os.mknod(path)
 
-        os.system("find /tmp/input-remapper-test/")
+        os.system(f"find {tmp}/")
```

Comment:
Oh, this will probably not work correctly if tmp has spaces in it now. Looks like it's just for debug logging but I should still fix that.

Reply:
Would changing this to `subprocess.call(["find", tmp])` be fine?

Reply:
oh, this is leftover from debugging. You can remove this line